.github/ISSUE_TEMPLATE/behavior.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Behavioral issue
+about: Use this template for issues where expected behavior does not match actual behavior, for example, a bug. Since this type of issue implicitly specifies a fix, it should be used for situations where there is existing agreement in the correct behavior. Use a general issue otherwise.
+title: <title describing problem and situation. e.g. accts.balance causes game to crash>
+labels: needs triage
+---
+
+### Repro Steps
+
+<!-- Provide steps to reproduce the issue that you are experiencing or seeing. Expected should describe the behavior you were expecting to observe. e.g. -->
+
+1. launch the game
+1. type accts.balance and hit enter
+
+**expected** the command completes successfully
+
+**actual** the game crashes
```

Comment:
Minor formatting gripe:
```suggestion
**Expected**: the command completes successfully

**Actual**: the game crashes
```
----
Alternatively, we could make these sub-headings, which would look better if expected and actual are larger (e.g. longer explanation text, screenshots or videos):
```suggestion
#### Expected
The command completes successfully

#### Actual
The game crashes
```